lib/services/presets.go

```diff
@@ -50,7 +50,8 @@ func NewPresetEditorRole() types.Role {
 				},
 			},
 			Allow: types.RoleConditions{
-				Namespaces: []string{apidefaults.Namespace},
+				Namespaces:            []string{apidefaults.Namespace},
+				DatabaseServiceLabels: types.Labels{types.Wildcard: []string{types.Wildcard}},
```

Comment:
Wouldn't it make more sense to have this as a part of the "access" role where all other labels are?

Reply:
Maybe both :thinking: 

So, the Discover User is the one that should have access to the DatabaseServiceLabels
I think that's part of the `editor` responsibilities (they are adding resources).
However, later on they will test the access, which is part of the `access` responsibilities.

I'll add it to the `access` role as well
Let me know if I should drop it from the `editor` role

Reply:
Yeah, I think it's similar to querying databases or servers or other resources - for the Discover flow the "access" role is what allows the user to do that, right? So I think let's remove it from here. You still need "access" role to be able to go through the Discover flow so you should have these labels.

Reply:
Yup, good point
Removing